frontend/HelperFuncs/utils.js

```diff
@@ -160,4 +169,51 @@ export const getRows = (transactions) => {
         }
     }
     return rows;
-};
\ No newline at end of file
+};
+
+
+/*
+helper function to check if a user's password meets the requirements of containing a special character, 
+upper and lower case letters, number, and length ≥ 8. If password passes all of these, return true and '' for the error message
+if false, return false and the appropriate error message. Returns are structured as a two-element list where index 0 contains
+the successs boolean and index 1 contains the string error message. 
```

Comment:
Good comment! I'm not in love with the 2 element list return value, the call site then requires indexing into the array, its not so clean. Perhaps an object would work just as well here, but it works.